src/main/main.ts

```diff
@@ -246,7 +242,23 @@ const openPathDialog = (event: any, args: any) => {
     console.log(err);
   })
 } 
+/**
+ * Checks the app config.
+ * @returns true if config is setup, false otherwise. 
+ */
+const checkConfig = () : boolean => {
+  if (mainWindow === null) return false; 
+  return isConfigReady(cfg) && (getLatestLog(baseLogPath));
+}
 
+/**
+ * Updates the status icon for the application.
+ * @param status the status number
+ */
+const updateStatus = (status: number) => {
+  if (mainWindow === null) return; 
```

Comment:
logic is a bit odd, how about 

`if (mainWindow !== null) mainWindow.webContents.send('updateStatus', status);`

Reply:
yeah we can do that, i just copied this from other places in the code.